src/pages/Home.vue

```diff
@@ -172,10 +176,14 @@ export default {
       //if params contain a searchString or pageNumber, save it into a variable,
       //else save the variable as undefined
       var searchString =
-        params != undefined && "searchString" in params ? params.searchString : undefined;
+        params != undefined && "searchString" in params
```

Comment:
<img width="423" alt="Screenshot 2021-07-13 at 1 16 04 PM" src="https://user-images.githubusercontent.com/13185699/125412378-7c245c00-e3dc-11eb-92be-81b8b5aba01c.png">
can you set the line limit within your Vue linter to 90? OR else it will keep making these changes which are not relevant!

Reply:
So the linter we use for vue files is called `vue 3 support - all in one`
You should install it from the extension store in vscode